### ISSUE TITLE
`Ctrl-O` prefixed commands not working in `jupyterlab>=4.1.0`

### ISSUE THREAD

**Issue by rpatel** [jupyterlab-contrib/jupyterlab-vim#138]
## Description
Vim command shortcuts that start with `Ctrl-O` (i.e., Cut Cell, Copy Cell, Paste Cell, Insert Cell Below, Insert Cell Above, Select First Cell, Select Last Cell, Center Cell) are not working since jupyterlab 4.1.0.

At the moment, I am requiring versions `jupyterlab==4.0.9` and `jupyterlab-vim==4.1.0` to keep these bindings working.

## Reproduce

1. Fresh install:  
`pip install 'jupyterlab>=4.1.0,<5.0.0a0' jupyterlab-vim`

2. In new notebook, any selected cell, edit mode, press `Ctrl-O, O` or `Ctrl-O, Ctrl-O`, or `Ctrl-O, G`, etc… the expected jupyter shortcut doesn't happen (i.e., insert cell below/above, select first cell).
  _However_, the editor does respond with the vim command mapped to the keys after the `Ctrl-O`s. For example, `Ctrl-O, O` inserts a new line and switches to insert mode, as if the `^O` prefix was never pressed.

## Context

- Python version 3.12.2
    - jupyterlab==4.1.2
    - jupyter_server==2.12.5
    - jupyterlab-vim==4.1.3
- Operating System and its version: CentOS 7
- Browser and its version: Chrome 121.0.6167.139

## 
Let me know if can provide any additional information.

**Comment by ianhi** [comment]
I can confirm this bug. @krassowski is there any chance this is related to the shortcut changes again? from https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#change-of-notebook-focus-handling-impacting-command-mode-shortcut-selectors

> Please note that :not(:read-write) fragment disables shortcuts when text fields (such as cell editor) are focused to avoid intercepting characters typed by the user into the text fields, however if your shortcut does not correspond to any key/typographic character (e.g. most shortcuts with Ctrl modifier) you may prefer to drop this fragment if you want the shortcut to be active in text fields.

we aren't using that tag, but maybe the auto conversion of things somehow got us?

**Comment by krassowski** [comment]
> we aren't using that tag, but maybe the auto conversion of things somehow got us?

I do not think so. It seems like the events are not properly intercepted so it might be the fallout from the change of the keydown capture mode in jupyterlab 4.1.

**Comment by rchavp** [comment]
Any update on this?

**Comment by madox2** [comment]
In the meantime, are there any workarounds?

**Comment by firai** [comment]
> In the meantime, are there any workarounds?

Unfortunately, I don't think so. I tested different possible substitute shortcuts, but most normal mode shortcuts will still clash with the default CodeMirror shortcuts one way or another.

All, please contribute to fixing https://github.com/jupyterlab/jupyterlab/issues/15897 if you're interested in helping.

**Comment by daeyun** [comment]
One possible workaround could be resetting `defaultKeymap` in a static .js file: https://github.com/jupyterlab/jupyterlab/issues/15897#issuecomment-2904965518

**Comment by daeyun** [comment]
If you want to disable CodeMirror shortcuts, https://github.com/jupyterlab/jupyterlab/issues/15897#issuecomment-2905083884
